lib/rules/react-no-multiple-render-methods.js

```diff
@@ -47,10 +45,19 @@ module.exports = {
 
         report(node);
       },
+      
+      ArrowFunctionExpression(node) {
```

Comment:
You are going to need to add a few tests for this. At the least I'd like to see it catch something like:
```
      code: `class Button extends React.Component {
        renderArrowFunction = () => {}
        renderBar = () => {}
        render() {}`
```

I think the error type will be different for these, so you will also need to modify the `error` utility to accept a `type` argument. Something like:

```
function error(memberName, type = 'MethodDefinition') {
  return {
    type,
    message: `Don’t use multiple render methods in a single component; they generally make your component harder to read. Instead break ${memberName} out into its own component and render it inside this one.`,
  };
}
```
then you should be able to do the following for the error of the above snippet:
```
errors: [
  error('renderArrowFunction', 'ArrowFunctionExpression'),
  error('renderBar', 'ArrowFunctionExpression'),
],
```

Happy to clarify anything you need.